service/src/main/java/org/broadinstitute/listener/relay/inspectors/TokenChecker.java

```diff
@@ -47,7 +47,7 @@ private Optional<Instant> tryDecodeAsB2cToken(String token) {
       var decoded = JWT.decode(token);
       return Optional.of(decoded.getExpiresAt().toInstant());
     } catch (com.auth0.jwt.exceptions.JWTDecodeException e) {
-      logger.info("Fail to check decode JWT", e);
+      logger.debug("Fail to check decode JWT", e);
```

Comment:
Shouldn't this be logger.error?

Reply:
I'm testing things out still...making it debug becuz I'm testing with google token..just to reduce some noise